app/src/main/java/com/stl/skipthelibrary/MyProfileActivity.java

```diff
@@ -1,26 +1,53 @@
 package com.stl.skipthelibrary;
 
 import androidx.appcompat.app.AppCompatActivity;
+import de.hdodenhof.circleimageview.CircleImageView;
```

Comment:
Out of curiosity is there a reason you're using this instead of just making an image circular with xml? is there something nice we get from it?